test/benchmarks/sails.load.test.js

```diff
@@ -205,10 +206,6 @@ function setupBenchmarks() {
  * @this {Array} benchmarks
  */
 function reportBenchmarks() {
-  var _ = require('lodash');
```

Comment:
Check it - https://github.com/balderdashy/sails/pull/3680/files/1e63bf2041ad27510ffca3b9695870cfc68f52d2#diff-a9884bc07f6f24626c2b107fa1fe4624L1